requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 hypothesis
 pyfrc~=2023.0.0b4
 pytest
+pre-commit
```

Comment:
I don't see any reason (another copy of) pre-commit should end up in a robot venv. It certainly shouldn't end up in (non-pre-commit) CI either.

Reply:
whats the recommended setup to install pre-commit?